recipe/meta.yaml

```diff
@@ -20,6 +20,61 @@ source:
 build:
   number: 0
   skip: True               # [py<37]
+  entry_points:
+    # distutils.commands
+    - alias=setuptools.command.alias:alias
```

Comment:
I think only [console_scripts](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/?highlight=console_scripts#console-scripts) have to be listed as entry points here.

Reply:
Yeah, see https://github.com/conda/conda-build/blob/78d2a4758be9a5b2e6c194b562ae6bd16092ade6/conda_build/utils.py#L1122. In conda, they are used to create console scripts. So all these will pollute the environment with random commands.